components/create_comment/create_comment.jsx

```diff
@@ -215,16 +236,9 @@ export default class CreateComment extends React.PureComponent {
         this.doInitialScrollToBottom = false;
     }
 
-    UNSAFE_componentWillMount() { // eslint-disable-line camelcase
+    componentDidMount() {
         this.props.clearCommentDraftUploads();
         this.props.onResetHistoryIndex();
-        this.setState({
-            draft: {...this.props.draft, uploadsInProgress: []},
```

Comment:
If we are removing these wouldn't this require the value to be set on either `constructor` or `getDerivedStateFromProps `?

Reply:
Have removed the setting of state.draft from the constructor completely that way in getDerivedStateFromProps if state.draft is undefined it can be set as it previously was in componentWillMount.  Does that sound OK?